<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVoice</class>
 <widget class="QWidget" name="QVoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpBox_global">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>191</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Global</string>
   </property>
   <widget class="QLabel" name="lbl_style">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Style :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_style">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Piano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Keys</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Organ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guitar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Synth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ethnic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bells</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rythm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SFX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtEdit_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="maximumBlockCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_feedback">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_feedback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Feedback :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_trans">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-128</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_trans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Transpose :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_porta_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Porta. time :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pmdctl_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PMD Control.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pitch_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pitch bender :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_poly_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mono / Poly :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_pitch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBut_poly">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Poly</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_pmdctl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Not assigned</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>After touch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mod. wheel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Breath ctl.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Foot ctl.</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spnBox_porta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBox_algos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorythm</string>
   </property>
   <widget class="QLabel" name="lbl_algo">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/imgs/algo1.bmp</pixmap>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_algo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBox_LFO">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>191</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>LFO</string>
   </property>
   <widget class="QSpinBox" name="spnBox_PMD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LFOwave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LFO wave :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_AMD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_LFOwave">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Saw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S&amp;H</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_AMD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AMD :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_PMD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PMD :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_LFOspeed">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LFOspeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LFO speed :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_PMS">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnBox_AMS">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_AMS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AMS :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_PMS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PMS :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_poly_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LFO load :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBut_load">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_poly_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LFO sync :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBut_sync">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sync</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
